tests/build.proj

```diff
@@ -27,7 +27,7 @@
 
   <Target Name="BatchRestorePackages" DependsOnTargets="VerifyDependencies">
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Restoring all packages..." />
-    
+
```

Comment:
I am not sure I follow - this file has no meanginful change, so we should revert it.